src/components/LoginForm/index.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import {
+  Form,
+  FormGroup,
+  TextInput,
+  Checkbox,
+  ActionGroup,
+  Button,
+  Title,
+} from '@patternfly/react-core';
+import { routerRedux } from 'dva/router';
+import { connect } from 'dva';
+import styles from './index.less';
+
+@connect(auth => ({
+  auth: auth.auth,
+}))
+class LoginForm extends React.Component {
```

Comment:
@gurbirkalsi I am not really sure why a login modal is required along with login page. It was discussed earlier in the design sync that it might enhance user experience. And the login modal itself reuses the `loginForm` but as discussed, we didn't want to include other login options and signup options in the modal so I added LoginForm component in the modal instead of Login Page component. Plus Login page component includes background image etc. which we do not want to be displayed in the modal.